src/product-card__button.css

```diff
@@ -0,0 +1,21 @@
+.product-card__button {
```

Comment:
use separate files only for blocks, not for elements. place all elements in their block's file